src/sql/parts/connection/common/serverInfoContextKey.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import { RawContextKey, IContextKeyService, IContextKey } from 'vs/platform/contextkey/common/contextkey';
+import { ServerInfo } from 'azdata';
+
+export class ServerInfoContextKey implements IContextKey<ServerInfo> {
+
+	static ServerInfoObj = new RawContextKey<ServerInfo>('serverInfoObj', undefined);
```

Comment:
Minor: we don't need this for now, as there's no ability to do . queries in context keys (e.g. `serverInfoObj.minorVersion`). Let's just remove for now as the name doesn't make sense either. I know ConnectionKey has something similar but we don't need it here.